spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfigurationTests.java

```diff
@@ -543,6 +545,48 @@ void shouldCloseDiscoveryClient() throws Exception {
 		assertThat(isShutdown.get()).isTrue();
 	}
 
+	@Test
+	void httpComponentsProtocolUpgradeRequestCustomizer_shouldNotCreateCustomRequestCustomizer() {
```

Comment:
Please follow the test naming convention present in this class, i.e., starting with `should...`, no need to prefix each test name with `httpComponentsProtocolUpgradeRequestCustomizer`.